mucommander-os-api/src/main/java/com/mucommander/desktop/DefaultDesktopAdapter.java

```diff
@@ -155,4 +156,46 @@ public boolean isApplication(AbstractFile file) {
     public ActionShortcuts getActionShortcuts() {
         return new ActionShortcuts();
     }
+
+    @Override
+    public boolean setIconBadgeNumber(int number) {
+        boolean opSuccessful = false;
+        if (Taskbar.isTaskbarSupported() &&
+                Taskbar.getTaskbar().isSupported(Taskbar.Feature.ICON_BADGE_TEXT)) {
+            if (number >= 0) {
+                Taskbar.getTaskbar().setIconBadge(Integer.toString(number));
+            } else {
+                Taskbar.getTaskbar().setIconBadge(null);
+            }
+            opSuccessful = true;
+        }
+        return opSuccessful;
+    }
+
+    @Override
+    public boolean setIconProgress(int progress) {
+        boolean opSuccessful = false;
+        if (Taskbar.isTaskbarSupported() &&
+                Taskbar.getTaskbar().isSupported(Taskbar.Feature.PROGRESS_VALUE)) {
+            if (progress >= 0 && progress <= 100) {
+                Taskbar.getTaskbar().setProgressValue(progress);
+            } else {
+                Taskbar.getTaskbar().setProgressValue(-1);
+            }
+            opSuccessful = true;
+        }
+        return opSuccessful;
+    }
+
+    @Override
+    public boolean requestUserAttention() {
+        boolean opSuccessful = false;
```

Comment:
how about inline this variable? the return value is already documented (same for those variables in the methods above)

Reply:
Matter of taste, I like having a variable that is being returned. If you insist I can change, I prefer the way it is.

Reply:
yeah, that's subjective but while I was trying to think of a better name to it, it occurred to me that removing it would be simpler.. the name `opSuccessful` suggests that the operation can complete successfully or fail while the value of this field rather indicates whether the operation was triggered - so I'd say it should be renamed to something like 'opSupportedByAwt' or 'awtActionTriggered'